indexers/keyvalue/MongoDBIndexer/tests/test_mongodbindexer.py

```diff
@@ -1,61 +1,85 @@
-import string
 import random
+from collections import Iterator
 
 import numpy as np
-from google.protobuf.json_format import MessageToJson
+from jina import Document
 
-from jina.drivers.helper import array2pb
-from jina.proto import jina_pb2, uid
 from .. import MongoDBIndexer
 
 
 def random_docs(num_docs, chunks_per_doc=5, embed_dim=10, jitter=1):
     c_id = 3 * num_docs  # avoid collision with docs
     for j in range(num_docs):
-        d = jina_pb2.Document()
-        d.tags['id'] = j
-        d.text = b'hello world doc id %d' % j
-        d.embedding.CopyFrom(array2pb(np.random.random([embed_dim + np.random.randint(0, jitter)])))
-        d.id = uid.new_doc_id(d)
-        for k in range(chunks_per_doc):
-            c = d.chunks.add()
-            c.text = 'i\'m chunk %d from doc %d' % (c_id, j)
-            c.embedding.CopyFrom(array2pb(np.random.random([embed_dim + np.random.randint(0, jitter)])))
-            c.tags['id'] = c_id
-            c.tags['parent_id'] = j
-            c_id += 1
-            c.parent_id = d.id
-            c.id = uid.new_doc_id(c)
+        with Document() as d:
+            d.tags['id'] = j
+            d.text = b'hello world doc id %d' % j
+            d.embedding = np.random.random([embed_dim + np.random.randint(0, jitter)])
+            for k in range(chunks_per_doc):
+                with Document() as c:
+                    c.text = 'i\'m chunk %d from doc %d' % (c_id, j)
+                    c.embedding = np.random.random([embed_dim + np.random.randint(0, jitter)])
+                    c.tags['id'] = c_id
+                    c.tags['parent_id'] = j
+                    c_id += 1
+                    c.parent_id = d.id
+                d.chunks.add(c)
         yield d
 
 
 def test_mongodbindexer():
-    keys = []
-    values = []
-    
     num_docs = 5
-    docs = list(random_docs(num_docs=num_docs, 
+    docs = list(random_docs(num_docs=num_docs,
                             chunks_per_doc=3))
-    keys = [uid.id2hash(doc.id) for doc in docs]
+    # adding type annotations to remove further warnings
+    keys: Iterator[int] = [doc.id for doc in docs]
     values = [doc.SerializeToString() for doc in docs]
-    
+
     query_index = random.randint(0, num_docs - 1)
-    query_id = docs[query_index].id
-    query_key = uid.id2hash(query_id)
+    # adding type annotations to remove further warnings
+    query_key: int = docs[query_index].id
```

Comment:
why is this annotation needed?

Reply:
`.query`, `.add` etc. expect `key` to be `int`, but `doc.id` is `UniqueId`. It's a warning from PyCharm, no problems at run time. Not sure why